packages/tensorboard/tensorboard.0.0.1/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+bug-reports:  "https://github.com/LaurentMazare/ocaml-tensorboard/issues"
+homepage:     "https://github.com/LaurentMazare/ocaml-tensorboard"
+dev-repo:     "git+https://github.com/LaurentMazare/ocaml-tensorboard.git"
+maintainer:   "Laurent Mazare <lmazare@gmail.com>"
+authors:      [ "Laurent Mazare" ]
+
+build: [["dune" "build" "--only-packages" "tensorboard" "--root" "." "-j" jobs "@install"]]
```

Comment:
Could you use the more standard `["dune" "build" "-p" name "-j" jobs]` if it's ok for you?

Reply:
Fixed, indeed this looks simpler.